test/integration/dynamic-esm/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This directory contains code that tests the ES6 syntax for importing and exporting code. ESLint needs a specific configuration to support this syntax, and we cannot use inline-configuration comments for this.